core/src/main/resources/jenkins/slaves/JnlpSlaveAgentProtocol3/description.jelly

```diff
@@ -1,4 +1,4 @@
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core">
-  ${%Extends the version 2 protocol by adding basic encryption but requires a thread per client}
+  ${%Extends the version 2 protocol by adding basic encryption but requires a thread per client.  This protocol falls back to Java Web Start Agent Protocol/2 (insecure) when it can't create a secure connection.}
```

Comment:
I would explicitly recommend to say this protocol is not recommended: https://github.com/jenkinsci/remoting/blob/master/docs/protocols.md#jnlp3-connect